openapi/components/schemas/GatewayAccountConfig/PayRedeem.yaml

```
@@ -0,0 +1,18 @@
+description: PayRedeem config
+allOf:
+  - $ref: ../GatewayAccount.yaml
+  - type: object
+    required:
+      - credentials
+    properties:
+      credentials:
+        type: object
+        description: PayRedeem credentials object
+        properties:
+          apiKey:
+            type: string
+            description: PayRedeem api key
```

nitpick: we should capitalize acronyms 
```suggestion
            description: PayRedeem API key.
```